book/src/simpleexample.md

```diff
@@ -145,3 +145,56 @@ eth_friend(?1, ?2, ?3, ?4) = and<
     Equal(?5["attestation"], ?3[?4])
 >
 ```
+
+## Another perspective
+When working with statements and operations, it might be useful to see them from another perspective:
+
+- A *statement* can be seen as the *constraints* of a traditional zk-circuit,
+  which can be true or false.
+- A *predicate* is a relation formula, which when filled with values becomes a
+  *statement*.
+- An *operation* comprises the deduction rules, which are rules used to deduce
+  new statements from previous statements or used to construct new statements
+  from values.
+
+$$
+predicate \cong circuit/relation to be fulfilled\\
+statement \cong constraints filled with the witness\\
+operations \cong deduction~rules
+$$
+
+
+For example,
+- `Equal` for integers is a predicate.
+- `5 Equal 5` is a statement (true)
+- `5 Equal 4` is a statement (false)
+
+
+
+<div style="display:flex;margin-left:-60px;">
+<div style="padding:10px;max-width:50%; border-right:1px solid #ccc;">
+    
+So, for example, for the given predicate:
+
+```
+IsWood(item, private: ingreds,inputs,key)=AND(
```

Comment:
I think using digital objects details in examples at this level feels like it will cause more confusion.  At this level nobody knows what ItemDef means.  I'd suggest using a simpler made-up example similar to the mathematical examples above.  Maybe a predicate like IsComposite(n) which proves a factorization using ProductOf statements?